uber/decorators.py

```diff
@@ -1,6 +1,19 @@
 from uber.common import *
 
 
+def catch_all_exceptions(func):
```

Comment:
A better name would be ``swallow_exceptions``, for two reasons:

1) That's the generic term of what we're doing.

2) You're actually not catching "all" exceptions.  You're only catching exception which extend ``Exception``, which is a subset of all exceptions.  For example, ``KeyboardInterrupt`` doesn't extend ``Exception``.  While this distinction is subtle and largely irrelevant to this function, it's still worth noting.  (If you did want to catch all exceptions you'd just say ``except:`` without specifying an exception class.